src/gpio.rs

```diff
@@ -397,6 +397,20 @@ macro_rules! gpio {
                     }
                 }
 
+                #[cfg(feature = "unproven")]
+                impl<MODE> InputPin for $PXx<Output<MODE>> {
```

Comment:
`impl` is missing for `PXx<Output<MODE>>` @ https://github.com/stm32-rs/stm32f3xx-hal/pull/105/files#diff-73928a829df74085c37fe73a4f3484fcL184-L206

`PXx` is a fully erased pin type, different from `$PXx`.

Reply:
Updated, I think.

Reply:
Updated, I think